python/py-imaplib2/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-imaplib2
+version             3.6
+revision            0
+
+categories-append   mail devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         A threaded Python IMAP4 client.
+long_description    ${description}
+
+homepage            https://github.com/jazzband/imaplib2/
+
+checksums           rmd160  2e8e893f8145f9a613c1d36c07007edec470132b \
+                    sha256  96cb485b31868a242cb98d5c5dc67b39b22a6359f30316de536060488e581e5b \
+                    size    26252
+
+python.versions     37 38 39
```

Comment:
Will this work with Python 3.10?

Reply:
@pmetzger good question. Let me try, and if it works => I'll switch `offlineimap` to 3.10